dappnode_package.json

```diff
@@ -5,7 +5,7 @@
   "upstreamRepo": "prysmaticlabs/prysm",
   "upstreamArg": "UPSTREAM_VERSION",
   "shortDescription": "Prysm prater ETH2.0 Beacon chain + validator",
-  "description": "Validate with prysm: a Go implementation of the Ethereum 2.0 Serenity protocol and open source project created by Prysmatic Labs. Beacon node which powers the beacon chain at the core of Ethereum 2.0\n\nIt includes a Grafana dashboard for the [DMS](http://my.dappnode/#/installer/dms.dnp.dappnode.eth) thanks to the amazing work of [metanull-operator](https://github.com/metanull-operator/eth2-grafana)",
+  "description": "Validate with prysm: a Go implementation of the Ethereum 2.0 Serenity protocol and open source project created by Prysmatic Labs. Beacon node which powers the beacon chain at the core of Ethereum 2.0\n\nIt includes a Grafana dashboard for the [DMS](http://my.dappnode/#/installer/dms.dnp.dappnode.eth) thanks to the amazing work of [metanull-operator](https://github.com/metanull-operator/eth2-grafana). This version of Prysm introduces a new method of authentication to access the Prysm WebUI.  The first time the package is started you will need to click on the link below next to the word token.  After opening the webUI with this token link you can access the webUI here at (prysm-prater.dappnode)[http://prysm-prater.dappnode] ",
```

Comment:
I would add the tip to use the URL link with the JWT (on the first login) in the `getting_started.md`, so it appears as higher as possible
![image](https://user-images.githubusercontent.com/41727368/142992501-a55fb97c-339d-4312-8585-358a97c87686.png)

Reply:
@pablomendezroyo Instructions have to be extremely clear on what to do. Consider using bold font or emojis to draw attention in a polite way

Reply:
check also https://github.com/dappnode/DAppNodePackage-prysm-prater/pull/22#discussion_r754978486

Reply:
Check https://github.com/dappnode/DAppNodePackage-prysm-prater/pull/22#issuecomment-976487501